riscv_analysis/src/cfg/function.rs

```diff
@@ -9,12 +9,27 @@ use super::CfgNode;
 
 #[derive(Debug, PartialEq, Eq)]
 pub struct Function {
-    pub nodes: Vec<Rc<CfgNode>>,
-    pub entry: Rc<CfgNode>, // Is only a FuncEntry node
-    pub exit: Rc<CfgNode>,  // Multiple exit points will be converted to
-    // a single exit point
+    /// Labels for the entry point of this function
+    labels: HashSet<With<LabelString>>,
+
+    /// List of all nodes in the function. May not be in any particular order.
+    nodes: RefCell<Vec<Rc<CfgNode>>>,
+
+    /// Entry node of the function.
+    pub entry: Rc<CfgNode>,
```

Comment:
In the spirit of making most fields private, why don't we also convert the member `.entry` to a function`.entry()` just like you did for `.exit`?